build-edt-k8s-agent.sh

```
@@ -1,59 +1,4 @@
-    --build-arg ONEC_USERNAME=$ONEC_USERNAME \
-    --build-arg ONEC_PASSWORD=$ONEC_PASSWORD \
-    --build-arg EDT_VERSION="$EDT_VERSION" \
-    --build-arg BASE_IMAGE=$BASE_IMAGE \
-    --build-arg BASE_TAG=$BASE_TAG \
-    --build-arg DOWNLOADER_REGISTRY_URL=$DOCKER_REGISTRY_URL \
-    --build-arg DOWNLOADER_IMAGE=oscript-downloader \
-    --build-arg DOWNLOADER_TAG=latest \
-    -t $DOCKER_REGISTRY_URL/edt:$edt_escaped \
-    -f edt/Dockerfile \
-    $last_arg
+./build-edt.sh
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)